### PR TITLE
Migrate commands to slash commands

### PR DESCRIPTION
**Pull request by agusmoles**
What was done:

- Created `/chitczat` and make it worked :joy:

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -1,23 +1,22 @@
...
 
-    if (!conversation.ok || conversation.channel.name != chatChannel) return;
+      if (chatChannel !== channel_name) return;
 
-    executeCommand(text, channel, user);
-  } catch (error) {
-    console.error(error);
-  }
-});
+      executeCommand(text, say, user_id);
```
Comment (kelostrada) [inline app.js:14]: The callback should be last param I guess, but it's fine :p

File: src/messages.js [hunk 2]
```diff
@@ -2,7 +2,7 @@ const { db } = require("./db");
 const { actions } = require("./actions");
 const { app, token } = require("./slack");
```
Comment (kelostrada) [inline src/messages.js:3]: Is it still required?

**Review by kelostrada — DISMISSED** [review]

**Review by kelostrada — APPROVED** [review]

**Comment by kelostrada** [comment]
❤️🧡💛💚💙💜

**Comment by agusmoles** [comment]
lol :heart:

**Pull request merged**